feature-libs/checkout/b2b/components/checkout-review-submit/checkout-review-submit.component.html

```diff
@@ -131,10 +131,21 @@ <h2 class="cx-review-title d-none d-lg-block d-xl-block">
   <!-- DELIVERY MODE SECTION -->
   <ng-template #deliveryMode>
     <div class="cx-review-summary-card cx-review-card-shipping">
-      <cx-card
-        *ngIf="deliveryMode$ | async as deliveryMode"
-        [content]="getDeliveryModeCard(deliveryMode) | async"
-      ></cx-card>
+      <div>
+        <cx-card
+          *ngIf="deliveryMode$ | async as deliveryMode"
+          [content]="getDeliveryModeCard(deliveryMode) | async"
+        >
+        </cx-card>
+        <ng-template
```

Comment:
I understand that a anew `div` tag is introduced. If this is a problem that might break customer styling, I can modify the SCSS styling for our component so that the `div` tag is not required.

However, please help me understand why a feature flag is needed in theHTML while introducing a new outlet?

Reply:
As we talked on the Slack, let's keep `ng-template` part, as they not rendering in the DOM, if the conditions are not met. Please just remove a `div` wrapper if it's possible.

Reply:
Will try removing the `div`, thanks @Matejk00.

Reply:
Hi @Matejk00,

While removing the `div` tag, the UI is rendered as `flex` with the direction as `row`.

In order to correct it, I would end up changing the parent style (removing the flex layout). However, it means that the customer's styling might be affected for `Delivery Mode`

![image](https://github.com/SAP/spartacus/assets/13379690/47c4b701-9ee2-4d40-833a-20abd9329ede)

Is there another alternative?